src/redux/reducers/index.ts

```diff
@@ -0,0 +1,13 @@
+// import all reducers
+import { combineReducers } from 'redux';
+import userStatsAPI from './userStatsAPI'
+import searchResultsAPI from './searchResultsAPI'
+import addSongToQueueAPI from './addSongToQueueAPI'
+
+const allReducers = combineReducers({
+  userStatsReducer : userStatsAPI,
```

Comment:
```
userStatsReducer: userStatsAPI
↑                 ↑
|                 Reducer name
|
property name in the state
```
Knowing this, it would make more sense to name them like

```
userStats: userStatsReducer
```

My personal preference would be `user` or `currentUser`.

Reply:
Also, if this is the place where you're supposed to store the access token, this is what the APIMiddleware is looking for when referring to `state.getState().authentication.acces_token`.